### PR TITLE
Feature/video layout

### PR DESCRIPTION
**Pull request by danbudris**
Bringing in the video layout elements so that we can begin to intergrate the fleshed-out video chat that Yergali built into the integrated frame.

### CODE REVIEW

File: group1/comm/static/comm_tool.css [hunk 1]
```diff
@@ -495,3 +505,37 @@ button#deleteButton {
...
+}
+
+.videoItems{
+  flex-grow: 1;
+  width: 100%;
+  height: 50%;
+
+}
+
+/* this is used to toole the hide for the video area */
```
Comment (gggoddardbu) [inline group1/comm/static/comm_tool.css]: This comment is not very clear.  Typo?

File: group1/comm/templates/comm/index.html [hunk 2]
```diff
@@ -96,14 +97,15 @@ <h4 class="sidebar_description">Quick links</h4>
...
+            <button id="startVideo" class="btn btn-default" type="button" data-toggle="tooltip" title="Start Video Share!" onclick="window.location.href='/communication/videochat'">
             <!-- 
             temporarily removed in order to fix a merge conflict; 10/29 DB
             <button id="sendcleartext" class="btn btn-default" type="button" data-toggle="tooltip" title="Start Video Share!" onclick="startVideoChat()"> 
             -->
                 <span class="glyphicon glyphicon-facetime-video" aria-hidden="true"></span>
             </button>
-            <div class="btn-group btn-group-inline">
+            
+	    <div class="btn-group btn-group-inline">
```
Comment (gggoddardbu) [inline group1/comm/templates/comm/index.html]: It looks like the spacing here might be slight off (1 or 2 spaces?).

File: group1/comm/static/comm_tool.css (hunk 1 above)
Comment (danbudris) [reply group1/comm/static/comm_tool.css]: fixed in 
8f75ba5

File: group1/comm/templates/comm/index.html (hunk 2 above)
Comment (danbudris) [reply group1/comm/templates/comm/index.html]: fixed in 
c61e863

**Pull request merged**